db/migrate/20181002123523_add_total_disk_size_to_physical_storages.rb

```diff
@@ -0,0 +1,5 @@
+class AddTotalDiskSizeToPhysicalStorages < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_storages, :total_disk_size, :bigint
```

Comment:
How about `:total_space` to stay consistent with `:storages`?

Reply:
@agrare You're right, it's better for consistency. I changed it to be `:total_space`.